### ISSUE TITLE
Python imports

### ISSUE THREAD

**Issue by ampli** [opencog/link-grammar#358]
Currently the following is used:
```
from linkgrammar import Sentence, Linkage, ParseOptions, Link, Dictionary
try:
    import linkgrammar._clinkgrammar as clg
except ImportError:
    import _clinkgrammar as clg
```
The **try** block is used so tests.py can work even before installing LG in the system.

This has the following drawbacks:
1. It is cumbersome.
2. It mentions a module starting with an underscore for no real need.
3. An IDE cannot check the usage of the clg.* functions, as _clinkgrammar is not Python code (it just flags as error all of them, e.g. `Module '_clinkgrammar' has no 'dictionary_set_data_dir' member`).

Drawbacks 2 and 3 can be solved by using **clinkgrammar** instead of **_clinkgrammar**. This is because SWIG also generates **clinkgrammar.py**, which imports **_clinkgrammar**.
However, drawbacks 1 still remains.

In order to make the usage of the Python interface more streamlined, I propose to "hide" the import of **_clinkgrammar**, as it is an implementation detail. This can be done by adding this in **linkgrammar.py**:
```
Clinkgrammar = clg
__all__ = ['ParseOptions', 'Dictionary', 'Link', 'Linkage', 'Sentence', 
           'Clinkgrammar']
```
(The last statement already exists - I only added to it `'Clinkgrammar' `.)

Then, in Python programs **only** the following is needed:
```
from linkgrammar import Sentence, Linkage, ParseOptions, Link, Dictionary, \
    Clinkgrammar as clg
```
Or even just:
```
import linkgrammar
```

I will send a PR for that (unless I missed something).

**Comment by linas** [comment]
Yes, hiding this stuff makes sense.

**Comment by ampli** [comment]
Implemented in #359.
